Add discussion on COUNT() with fill()

rkuchan opened the pull request:
I added a section about `fill()` with functions (in general) which links back to the Data Exploration page. 

I also added a section that discusses the specific behavior of `COUNT()` with `fill()`. This fixes https://github.com/influxdb/docs.influxdata.com/issues/96.

beckettsean commented:
+1

Not sure when we are switching to 0.10 for the docs. @gunnaraasen should we be making those changes now? Seems like we can't launch the docs until late Jan if these changes are in place, or are we documenting 0.10 before its release?

rkuchan commented:
I changed all the links back to v0.9 and I also added a short section on including multiple functions in a single query. That last change fixes https://github.com/influxdb/docs.influxdata.com/issues/116.

beckettsean commented:
+1 LGTM
